src/main/java/gyro/google/compute/InstanceNetworkInterface.java

```diff
@@ -71,24 +77,28 @@ public void setNetworkIp(String networkIp) {
 
     /**
      * A list of access configurations for this interface. Currently, only ``NE_TO_ONE_NAT`` is supported. If unspecified this instance will have no external internet access.
+     *
+     * @subresource gyro.google.compute.InstanceAccessConfig
      */
-    public List<InstanceAccessConfig> getAccessConfigs() {
-        return accessConfigs;
+    public List<InstanceAccessConfig> getAccessConfig() {
+        return accessConfig;
     }
 
-    public void setAccessConfigs(List<InstanceAccessConfig> accessConfigs) {
-        this.accessConfigs = accessConfigs;
+    public void setAccessConfig(List<InstanceAccessConfig> accessConfig) {
+        this.accessConfig = accessConfig;
     }
 
     /**
      * A list of alias IP ranges for this network interface. Can only specify this for network interfaces in VPC networks.
+     *
+     * @subresource gyro.google.compute.InstanceAliasIpRange
      */
-    public List<InstanceAliasIpRange> getAliasIpRanges() {
-        return aliasIpRanges;
+    public List<InstanceAliasIpRange> getAliasIpRange() {
+        return aliasIpRange;
```

Comment:
To be consistent with how we treat `Collection` type attributes.
This enables us to use the getter without doing NPE checks

```suggestion
        if (aliasIpRange == null) {
                aliasIpRange = new ArrayList<>();
        }
        
        return aliasIpRange;
```